auto_rx/autorx/templates/index.html

```diff
@@ -311,15 +311,15 @@
             });
 
             // Check if user has preffered map visiblity.
-            if (getCookie('map') == 'true') {
+            if (getCookie('map') == 'true' || getCookie('map') == null) {

```

Comment:
This should work. A simpler alternative may be to reverse the condition and check for `'false'`, as is done for some other settings.

Reply:
Good idea. Corrected in the below commit.